src/corelibs/U2Formats/src/FastqFormat.cpp

```diff
@@ -478,30 +544,30 @@ DNASequence *FastqFormat::loadSequence(IOAdapter* io, U2OpStatus& os) {
     QString sequenceName = readSequenceName(os, io, '@');
     // check for eof while trying to read another FASTQ block
     CHECK_EXT(!io->isEof(), , NULL);
-    CHECK_OP(os, NULL);
+    CHECK_OP(os, new DNASequence());
```

Comment:
Why do you want to return "new DNASequence()" instead of NULL?
There is a code that expects exactly a sequence (StreamSequenceReader::hasNext()). What should we do with it?